### DISCUSSION TITLE
nested hasmanyrepeater

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2249] [Help]
I have a hasmanyrepeater relationship inside another, and at the time of insert it does it well, but when displaying in the edit, it does not return the second hasmanyrepeater that is inside the other, is it possible to correct it?

Forms\Components\HasManyRepeater::make('ProductItemColor')
                        ->relationship('ProductItemColor')->schema([

  Forms\Components\HasManyRepeater::make('ProductItemColorSizes')
                              ->relationship('ProductItemColorSizes')->schema([
  
  ])

])

**Comment by danharrin** [comment]
Make sure you're on the latest version, this was recently fixed

**Reply by jcgmalave** [reply]
i just run php artisan filament:upgrade and composer update and still its not working :(

**Reply by danharrin** [reply]
Please open an issue with a GitHub repo I can use to test

**Reply by jcgmalave** [reply]
Done, it was due to the upgrade, thanks a lot
